sbin/build.sh

```diff
@@ -387,6 +387,11 @@ buildTemplatedFile() {
     ADDITIONAL_MAKE_TARGETS=" test-image"
   fi
 
+  if [[ "${BUILD_CONFIG[MAKE_EXPLODED]}" == "true" ]]; then
+    # In order to make an exploded image we cannot have any additional targets
+    ADDITIONAL_MAKE_TARGETS=""
```

Comment:
Same goes for the previous comment, have we bound this as late as possible?

Reply:
this is the latest place it can be set